deno/payloads/v10/channel.ts

```diff
@@ -1476,16 +1512,52 @@ export interface APIAttachment {
 	 * Attachment flags combined as a bitfield
 	 */
 	flags?: AttachmentFlags;
+	/**
+	 * For Clips, array of users who were in the stream
+	 */
+	clip_participants?: APIUser[];
```

Comment:
Can you make sure that for endpoints that can post attachments, this won't cause issues 🙏

Reply:
those use

https://github.com/discordjs/discord-api-types/blob/f1257b1a6a59df2bf0afa1d0f8b548a4fd36517c/rest/v10/channel.ts#L266-L273

so they should not be affected